src/components/Reown/DAppInfo.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { t } from 'ttag';
+
+/**
+ * Reusable component for displaying dApp information
+ * Shows dApp icon, proposer name, and URL/chain info
+ */
+export const DAppInfo = ({ dapp, className = "d-flex align-items-center mb-4" }) => (
```

Comment:
suggestion: Have `className` to be empty by default.

That way, when a caller needs to clear its classes, it's not mandatory to send an empty parameter.

From what I've seen, only `BaseNanoContractModal.js`  calls this component without passing its classes. So it's the only call that will need to be changed to have `className = "d-flex align-items-center mb-4"`

Reply:
Hey, instead of setting the default to empty, I've changed it to `d-flex align-items-center mb-4`, this way 4 out of 5 components can just not pass the props. What do you think?